pkg/services/ngalert/state/state_tracker.go

```diff
@@ -75,19 +80,25 @@ func (st *StateTracker) set(stateEntry AlertState) {
 	st.stateCache.cacheMap[stateEntry.CacheId] = stateEntry
 }
 
-func (st *StateTracker) get(stateId string) AlertState {
+func (st *StateTracker) Get(stateId string) AlertState {
```

Comment:
This is now exported, primarily for testing, but it's also reasonable for getters/setters to be accessible so...